libyul/AST.h

```diff
@@ -38,26 +38,28 @@ using Type = YulString;
 
 struct DebugData
```

Comment:
Should we move this to ``<liblangutil/DebugData.h>`` now? Then we'd get around the ``yul::`` in libevmasm - and it's no longer really yul specific now.